fe/fe-core/src/main/java/com/starrocks/lake/LakeTablet.java

```diff
@@ -55,13 +56,19 @@ public class LakeTablet extends Tablet {
     private static final String JSON_KEY_DATA_SIZE = "dataSize";
     private static final String JSON_KEY_ROW_COUNT = "rowCount";
     private static final String JSON_KEY_DATA_SIZE_UPDATE_TIME = "dataSizeUpdateTime";
+    private static final String JSON_KEY_PK_INDEX_MEM_SIZE = "primaryIndexMem";
+    private static final String JSON_KEY_PK_INDEX_DISK_SIZE = "primaryIndexDisk";
 
     @SerializedName(value = JSON_KEY_DATA_SIZE)
     private volatile long dataSize = 0L;
     @SerializedName(value = JSON_KEY_ROW_COUNT)
     private volatile long rowCount = 0L;
     @SerializedName(value = JSON_KEY_DATA_SIZE_UPDATE_TIME)
     private volatile long dataSizeUpdateTime = 0L;
+    @SerializedName(value = JSON_KEY_PK_INDEX_MEM_SIZE)
```

Comment:
In my opinion, the primary index size is also useful for normal tablets (LocalTablets), isn't it?